metatron-app/metatron-discovery
# F#2298 generate metadata when datasource enabled

### Description
Changes metadata creation time since creation of datasource

**Related Issue** : #2298 


### How Has This Been Tested?
1. create DataSource normally.
2. The "Go to Metadata" button should not be present immediately after creation.
3. The "Go to Metadata" button should appear after the completion message.

1. create DataSource again (for error)
2. To create a DataSource ingestion error, add the column using the wrong longitude, latitude column.
3. The "Go to Metadata" button should not be present immediately after creation.
4. DataSource should not appear in metadata list after error occurs.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
